doc/src/features/ExcelImport.md

```diff
@@ -0,0 +1,31 @@
+# Import NWKs from Excel
+
+A button labeled `Import` makes it possible to import NWKs from an Excel file.
+
+### Using the button
+
+When the button is clicked, a dialog will open.
+
+It is possible to upload an Excel file to the field in the middle of the dialog.
+
+It is not possible to upload non-Excel files or old format Excel files (older than 2007).
+
+The Excel file has to be in the <a download="Import_Standart.xlsx" href="assets/Import_Standart.xlsx">correct format</a>.
```

Comment:
when i download an open the file I get an error:

![grafik](https://github.com/it-at-m/Praktikumsplaner/assets/13592751/f4c2f23e-8d9d-4259-9800-012b2ef8d9b4)

Reply:
maybe a local Problem. When i download the File from the branch i can open it succefully.